### ISSUE TITLE
Screenreader doesn't announce links correctly on homepage

### ISSUE THREAD

**Issue by carodew** [18F/atf-eregs#516]
On the homepage, under "Legal limitations of this tool," VoiceOver doesn't announce the links in the paragraph text correctly, but only announces the `aria-labelledby` text. It should also announce the destination of the link. 

## Steps – 
1. activate VoiceOver on your Mac by pressing Command+F5
2. click on the heading "Legal limitations of this tool" in the webpage to shortcut to the right area of the website
3. tab until you get to the links in the paragraph under the heading "Legal limitations of this tool" 
4. VoiceOver will announce "Link, link opens in new window." 
5. To turn off VoiceOver, press Command+F5 again

Voiceover must announce the link text, and ideally announce that the link opens in a new window (but this is secondary to announcing the link text). So the result should be that VoiceOver announces "Link, Code of Federal Regulations" or "Link, Code of Federal Regulations, opens in a new window."

I think you can have it announce both using `aria-labelledby` and an ID on the link, however ARIA can be complicated and isn't well supported by screen readers. Another option to try (and one that is probably more robust) is to add appropriate alternative text to the new window icon. 

More – 
- https://www.w3.org/TR/WCAG20-TECHS/ARIA7.html
- http://webaim.org/techniques/hypertext/hypertext_links#new_window

**Comment by carodew** [comment]
Update – this may be true for all external links across the site.

**Comment by cmc333333** [comment]
Should be resolved once we pull in upstream: eregs/regulations-site#502
